plugin/src/main/java/com/denizenscript/denizen/objects/properties/material/MaterialDisarmed.java

```diff
@@ -0,0 +1,103 @@
+package com.denizenscript.denizen.objects.properties.material;
+
+import com.denizenscript.denizen.objects.MaterialTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.properties.Property;
+import com.denizenscript.denizencore.objects.properties.PropertyParser;
+import org.bukkit.Bukkit;
+import org.bukkit.block.data.BlockData;
+import org.bukkit.block.data.type.Tripwire;
+
+public class MaterialDisarmed implements Property {
+
+    public static boolean describes(ObjectTag material) {
+        if (!(material instanceof MaterialTag)) {
+            return false;
+        }
+        MaterialTag mat = (MaterialTag) material;
+        if (!mat.hasModernData()) {
+            return false;
+        }
+        BlockData data = mat.getModernData();
+        return data instanceof MaterialTag
+                || ((MaterialTag) material).getModernData() instanceof Tripwire;
+    }
+
+    public static MaterialDisarmed getFrom(ObjectTag _material) {
+        if (!describes(_material)) {
+            return null;
+        }
+        else {
+            return new MaterialDisarmed((MaterialTag) _material);
+        }
+    }
+
+    public static final String[] handledMechs = new String[] {
+            "disarmed"
+    };
+
+    private MaterialDisarmed(MaterialTag _material) {
+        material = _material;
+    }
+
+    public MaterialTag material;
+
+    public static void register() {
+
+        // <--[tag]
+        // @attribute <MaterialTag.disarmed>
+        // @returns ElementTag
+        // @mechanism MaterialTag.disarmed
+        // @group properties
+        // @description
+        // Returns the current disarmed state of the tripwire.
+        // For tripwires, is TRUE (corresponding to "disarmed") or FALSE (corresponding to "armed").
+        // -->
+        PropertyParser.registerStaticTag(MaterialDisarmed.class, ElementTag.class, "disarmed",
+                (attribute, material) -> new ElementTag(material.getDisarmable().isDisarmed()),"is_disarmed");

```

Comment:
use `{}` on lambdas.

Also get rid of that... brand newly added deprecated name variant? 0.o